pyproject.toml

```diff
@@ -36,13 +36,22 @@ dependencies = [
     "tabulate>=0.8.3", # pandas optional dependency for .to_markdown()
     "imagehash>=4.2.0",
     "tqdm>=4.53.0",
-    "matplotlib>=3.4"
+    "matplotlib>=3.4",
+    "fsspec==2023.1.0; python_version<'3.8'", # latest compatible with Python 3.7
```

Comment:
Why the latest compatible and not the minimum compatible version here?

Reply:
Could you explain the difference? 
What I meant is that later versions do not support python 3.7. If the user's python version is above 3.7, it's installing the minimum compatible version.